### DISCUSSION TITLE
Would Netdata be a good replacement for Nagios etc. for large scale network monitoring?

### DISCUSSION THREAD

**Discussion by tphakala** [netdata/netdata#17229] [Q&A] [answered]
Coming from managed networks industry point of view, would Netdata be a good fit for monitoring LAN & WAN network devices such as LAN switches, routers, firewalls, etc. using methods like ping, SNMP, REST-API etc?

**Comment by ilyam8** [comment, marked as answer]
Hi, @tphakala. No, Netdata is not good for network monitoring. It's for servers/applications monitoring, not network devices.

Nice profile picture btw

**Reply by Ferroin** [reply]
To expand on this a bit, Netdata _can_ do some degree of monitoring of this type of stuff (we have some rather bare-bones support for SNMP, as well as basic stuff like checking HTTP endpoints or pinging remote hosts), but it is very much not designed with this as a focus, and as a result the Netdata Agent is far less capable (and far less user-friendly) for this type of thing than a proper network monitoring tool like Nagios or Icinga.

**Comment by mcdent** [comment]
I’ve started trying some basic ping and port connections. The main thing which jumps out at me is the lack of seeing the names of the hosts you are monitoring. I think there’s a work around but even then the next is so small it’s not easy to identify the individual devices you are monitoring at a glance.
